tools/dynamatic/scripts/simulate.sh

```diff
@@ -69,9 +69,8 @@ echo_info "Launching Modelsim simulation"
 cd "$HLS_VERIFY_DIR"
 "$HLS_VERIFIER_BIN" \
   --resource-path="$RESOURCE_DIR" \
-  --ctb-path="../C_SRC/$KERNEL_NAME.c" \
-  --cduv-path="../C_SRC/$KERNEL_NAME.c" \
   --cfuv-function-name="$KERNEL_NAME" \
-  --hdl-duv-entity-name="$KERNEL_NAME"_wrapper \
+  --hdl-duv-entity-name="$KERNEL_NAME" \
```

Comment:
But I can't tell how you distinguished between these two names in the code.
Sometimes I thought "This should be the hdl name, but the cfuv name is used instead."
Could you just unify these two arguments?